federation/pkg/kubefed/init/init_test.go

```diff
@@ -889,6 +894,11 @@ func fakeInitHostFactory(apiserverServiceType v1.ServiceType, federationName, na
 	sort.Strings(apiserverArgs)
 	apiserverCommand = append(apiserverCommand, apiserverArgs...)
 
+	nodeSelector, err := marshallOverrides(nodeSelectorString)
+	if err != nil {
+		return nil, fmt.Errorf("error marshalling --node-selector: %v", err)
```

Comment:
@gyliu513 This is not a testcase, just a fake method. `t.Fatalf` is not working here.